src/main/java/bowling/domain/Score.java

```diff
@@ -0,0 +1,62 @@
+package bowling.domain;
+
+import java.util.Objects;
+
+class Score {
+    static final int MAX = 10;
```

Comment:
`Score` 외에 사용하는 곳이 없어 보이네요. private으로 바꾸면 어떨까요?

```suggestion
    private static final int MAX = 10;
```

Reply:
🆗